app/src/routes/v1/ipc.js

```diff
@@ -1,37 +1,13 @@
 const router = require('express').Router();
-
-const Constants = require('../../components/constants');
 const log = require('npmlog');
 const Problem = require('api-problem');
-const validation = require('../../middleware/validation');
 
-const db = require('../../models');
+const validation = require('../../middleware/validation');
+const dataService = require('../../services/dataService');
```

Comment:
i guess we're not using a backend component for handing the ipc related stuff at this stage.
so calling the data service directly from the route?

Reply:
Yes - for now most of that persistence logic will be in the data layer, so the route layer should not be needing to ask for the db models. Once we get more clarity on business logic requirements, we can change that behavior if necessary.